modules/eks/security_groups.tf

```diff
@@ -0,0 +1,209 @@
+locals {
```

Comment:
Remove this file and any reference to the resources created here. SG created by EKS is sufficient enough.